src/proposals/20231115/SparkEthereum_20231115.t.sol

```diff
@@ -0,0 +1,236 @@
+
+        /*****************************/
+        /*** WBTC After Assertions ***/
+        /*****************************/
+
+        ReserveConfig memory wbtcConfigAfter = _findReserveConfigBySymbol(allConfigsAfter, 'WBTC');
+
+        ReserveConfig memory wbtcConfigExpected = ReserveConfig({
+            symbol:                  'WBTC',
+            underlying:               _findReserveConfigBySymbol(allConfigsBefore, 'WBTC').underlying,
+            aToken:                   address(0),  // Mock, as they don't get validated in '_validateReserveConfig'
+            variableDebtToken:        address(0),  // Mock, as they don't get validated in '_validateReserveConfig'
+            stableDebtToken:          address(0),  // Mock, as they don't get validated in '_validateReserveConfig'
+            decimals:                 8,
+            ltv:                      70_00,
+            liquidationThreshold:     75_00,
+            liquidationBonus:         107_00,
+            liquidationProtocolFee:   10_00,
+            reserveFactor:            20_00,
+            usageAsCollateralEnabled: true,
+            borrowingEnabled:         true,
+            interestRateStrategy:     wbtcConfigAfter.interestRateStrategy,
+            stableBorrowRateEnabled:  false,
+            isPaused:                 false,
+            isActive:                 true,
+            isFrozen:                 false,
+            isSiloed:                 false,
+            isBorrowableInIsolation:  false,
+            isFlashloanable:          true,
+            supplyCap:                3_000,
+            borrowCap:                2_000,
+            debtCeiling:              0,
+            eModeCategory:            0
+        });
+        _validateReserveConfig(wbtcConfigExpected, allConfigsAfter);
+        _validateInterestRateStrategy(
+            wbtcConfigAfter.interestRateStrategy,
+            wbtcConfigExpected.interestRateStrategy,
+            InterestStrategyValues({
+                addressesProvider:             address(poolAddressesProvider),
+                optimalUsageRatio:             0.60e27,
+                optimalStableToTotalDebtRatio: 0,
+                baseStableBorrowRate:          0.02e27,
+                stableRateSlope1:              0,
+                stableRateSlope2:              0,
+                baseVariableBorrowRate:        0,
+                variableRateSlope1:            0.02e27,
+                variableRateSlope2:            3.00e27
+            })
+        );
+        assertTrue(wbtcConfigAfter.aToken            != address(0));
+        assertTrue(wbtcConfigAfter.variableDebtToken != address(0));
+        assertTrue(wbtcConfigAfter.stableDebtToken   != address(0));
+    }
+
+    function testCrossChainExecution() public {
+        GovHelpers.executePayload(vm, payload, executor);
+
+        gnosis.relayFromHost(true);
+        skip(2 days);
+
+        IL2BridgeExecutor(GNOSIS_BRIDGE_EXECUTOR).execute(1);
```

Comment:
Is there any state assertions we want to add here?

Reply:
For now I assumed not reverting is the minimum. The logic is tested in the Gnosis spell test file, but I can copy these tests here as well if you think that makes sense.

Reply:
Perhaps an `expectCall()` or something.

Reply:
I won't block deployment on this but yeah I think itd be really nice to assert gnosis changes within this change, @barrutko and I can discuss offline.

Reply:
Responded below.